src/wp-includes/js/media/views/attachments.js

```diff
@@ -1,26 +1,54 @@
 /**
  * wp.media.view.Attachments
  *
- * @class
- * @augments wp.media.View
- * @augments wp.Backbone.View
- * @augments Backbone.View
+ * Represents the overview of attachments in the Media Library.
+ */
+
+/**
+ * @access private
  */
 var View = wp.media.View,
 	$ = jQuery,
 	Attachments;
 
-Attachments = View.extend({
+Attachments = View.extend( /** @lends wp.media.view.Attachments.prototype */ {
 	tagName:   'ul',
 	className: 'attachments',
 
 	attributes: {
 		tabIndex: -1
 	},
 
+	/**
+	 * Binds events to the collection this view represents when adding or removing attachments or resetting the entire collection.
+	 * Binds events to scrolling to trigger the scroll function.
+	 *
+	 * @constructs
+	 * @memberOf wp.media.view
+	 *
+	 * @augments wp.media.View
+	 * @augments wp.Backbone.View
+	 * @augments Backbone.View
+	 *
+	 * @listens collection:add
+	 * @listens collection:remove
+	 * @listens collection:reset
+	 * @listens controller:library:selection:add
+	 * @listens scrollElement:scroll
+	 * @listens this:ready
+	 * @listens controller:open
+	 */
 	initialize: function() {
 		this.el.id = _.uniqueId('__attachments-view-');
 
+		/*
+		 * refreshSensitivity The time in milliseconds to throttle the scroll handler.
+		 * refreshThreshold The threshold that should be scrolled before loading more attachments from server.
```

Comment:
What does this threshold number mean? seconds? Please explain the number, instead of using `the threshold`, so for example: `The number of seconds that should be scrolled before...`
And: `from *the* server`